ui/vty/src/Ariadne/UI/Vty/Widget/Wallet.hs

```diff
@@ -177,7 +193,14 @@ drawWalletWidget focus WalletWidgetState{..} = do
     B.padAll 1 $
     B.vBox $
     padBottom <$>
-      [ label "Wallet name:" B.<+> B.txt walletName
+      [ label "Wallet name:"
+          B.<+> drawChild WidgetNameWalletName
+          B.<+> padLeft (drawChild WidgetNameWalletRenameButton)
+      , case walletRenameResult of
+          RenameResultNone -> B.emptyWidget
```

Comment:
Maybe we will consider abstracting something for widgets that perform backend commands in the future...

Reply:
Yeah, I thought about that too.
That's also the reason I try to make this code as similar looking as possible, so that it can be easily and mechanically refactored in the future.